.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  semi: true,
+  trailingComma: "all",
+  singleQuote: true,
+  printWidth: 120,
+  tabWidth: 2
+};
```

Comment:
do we need a newline here

Reply:
good catch! it's always great to have another pair of eyes another pair of eyes another pair of eyes eyse another eyes

Reply:
👀 👀 👀 👀 👀 👀 👀 👀 👀 👀 👀 👀 👀